TypeCobol.Test/Parser/Programs/TypeCobol/TypePicNoChild.rdzMix.txt

```diff
@@ -40,12 +39,10 @@ Line 33[8,23] <27, Error, Syntax> - Syntax error : Elementary item USAGE toto2 s
          05 var3 pic 9.
 
       *KO because var1 contains a child
```

Comment:
Can you change the "KO" by "Should be KO, but outside the scope of this issue"

This also apply to several others "KO"